go.mod

```diff
@@ -2,4 +2,12 @@ module github.com/omec-project/pfcpsim
 
 go 1.16
 
-require github.com/wmnsk/go-pfcp v0.0.14
+require (
+	github.com/antoninbas/p4runtime-go-client v0.0.0-20211006214122-ea704d54a7d3
```

Comment:
Could you run `go mod tidy` ? I believe that `p4runtime-go-client` shouldn't be imported.